bin/yaml/hlsl.yaml

```diff
@@ -12,3 +12,15 @@ compilers:
         targets:
           - name: trunk
             compiler_name: dxc
+    rga:
+      type: tarballs
+      url: https://github.com/GPUOpen-Tools/radeon_gpu_analyzer/releases/download/{name}/rga-linux-{name}.tgz
+      compression: gz
+      dir: rga-{name}.{extra_number} # The folder inside has an additional build number
+      check_exe: rga --version
+      strip_components: 1
```

Comment:
Nice find :) At one stage I thought all this YAML was a "simple" thing but...it's so complex!